pkg/util/util.go

```diff
@@ -514,3 +515,37 @@ func ReadDirNoStat(dirname string) ([]string, error) {
 
 	return f.Readdirnames(-1)
 }
+
+func SpliceErrors(errors interface{}) string {
+	t := reflect.TypeOf(errors)
+	var errStrings []string
+	switch t.Kind() {
+	case reflect.Slice, reflect.Array:
+		v := reflect.ValueOf(errors)
+		for i := 0; i < v.Len(); i++ {
+			if err, ok := v.Index(i).Interface().(error); ok {
+				errStrings = append(errStrings, err.Error())
+			}
+		}
+	default:
+		if err, ok := reflect.ValueOf(errors).Interface().(error); ok {
+			errStrings = append(errStrings, err.Error())
+		}
+	}
+
+	if len(errStrings) == 0 {
+		return ""
+	}
+
+	if len(errStrings) == 1 {
+		return errStrings[0]
+	}
+
+	var stb strings.Builder
+	stb.WriteString("[\n")
+	for _, errString := range errStrings {
+		stb.WriteString(fmt.Sprintf("  %s\n", errString))
+	}
+	stb.WriteString("]\n")
+	return stb.String()
+}
```

Comment:
```suggestion
func SpliceErrors(errors []error) string {
	if len(errStrings) == 0 {
		return ""
	}

	var stb strings.Builder
	stb.WriteString("[\n")
	for _, err := range errors {
		stb.WriteString(fmt.Sprintf("  %s\n", err.Error()))
	}
	stb.WriteString("]\n")
	return stb.String()
}
```
The errors in the parameter can only be slice.